wordpress-mobile/WordPress-Android
# Issue/update reader analytics

### Fix
Update analytics in the ***Reader*** tab.  One [event](https://github.com/wordpress-mobile/WordPress-Android/compare/issue/update-reader-analytics?expand=1#diff-bd4eaa15deb1338da7ce24981478314dL842) was incorrectly tracking likes when the post was unliked.  

### Test
0. Set breakpoints on events.
1. Go to ***Reader*** tab.
2. Tap ***View*** button on post card.
3. Notice ***READER_ARTICLE_VIEWED*** is hit.
4. Tap ***Like*** button on post card.
5. Notice ***READER_ARTICLE_LIKED*** is hit.
6. Tap ***Like*** button on same post card.
7. Notice ***READER_ARTICLE_UNLIKED*** is hit.
8. Tap post content to go to detail view.
9. Tap ***Like*** button.
10. Notice ***READER_ARTICLE_LIKED_DETAIL*** is hit.
11. Tap ***Like*** button.
12. Notice ***READER_ARTICLE_UNLIKED_DETAIL*** is hit.
